Document/Document_files/cm1.html

```diff
@@ -17,8 +17,8 @@
 <h1 style="border-bottom:1px solid black;text-align:center;">PyCM Report</h1>
 <h2>Dataset Type : </h2>
 <ul>
-<li>Multi-Class Classification</li>
 <li>Balanced</li>
+<li>Multi-Class Classification</li>
```

Comment:
this is a bit odd. Why did that happen?

Reply:
There is no issue. 
This is related to https://github.com/sepandhaghighi/pycm/pull/592 following the HTML functions refactor.